src/main/kotlin/no/nav/syfo/util/FormdataToFagmelding.kt

```diff
@@ -0,0 +1,144 @@
+package no.nav.syfo.util
+
+import no.nav.helse.op2016.Skjemainnhold
+import no.nav.syfo.service.domain.*
+
+fun mapFormdataToFagmelding(
+    fnr: String,
+    skjemainnhold: Skjemainnhold,
+): Fagmelding = Fagmelding(
+    oppfolgingsplan = Oppfolgingsplan(
+        nokkelopplysninger = nokkelopplysninger(
+            skjemainnhold,
+        ),
+        opplysningerOmArbeidstakeren = opplysningerOmArbeidstakeren(
+            fnr,
+            skjemainnhold,
+        ),
+        opplysingerOmSykefravaeret = opplysingerOmSykefravaeret(
+            skjemainnhold
+        ),
+        tiltak =  tiltak(
+            skjemainnhold
+        ),
+        arbeidstakersDeltakelse = arbeidstakersDeltakelse(
+            skjemainnhold
+        ),
+        utfyllendeInfo = skjemainnhold.utfyllendeOpplysninger
+    )
+)
+
+fun nokkelopplysninger(
+    skjemainnhold: Skjemainnhold,
+) = Nokkelopplysninger(
+    virksomhetensnavn = skjemainnhold.arbeidsgiver.orgnavn,
+    organiasjonsnr = skjemainnhold.arbeidsgiver.orgnr,
```

Comment:
typo: organiasjonsnr -> organisasjonsnr

Reply:
@AudunSorheim Disse variabelnavnene (med åpenbare trykkleifer) brukes / referes også i `syfooppdfgen`, så da må man forandre begge steder. Tenker det kan taes i en senere PR for å unngå at scope og testomfang øker ytterligere 😅